src/gdash/scripts/_cloneArray/_cloneArray.gml

```diff
@@ -9,13 +9,10 @@ _isEqual(myArray, copyArray)
 // => true
 */
 
-var inputArray = argument0;
+// Abuse GML array copy into script mechanics.
+/// Note, maybe switch to GMS2 new method: array_copy?

```

Comment:
Yeah, I'd glad to see `array_copy` get primary support from GML in GM:S2, but `_cloneArray` is still useful for them GM:S1 folks. Will keep this comment to encourage people to upgrade :)